### PR TITLE
Implemented code for static UI of settings page (#1vju6bh)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
desktop-settings

mobile-settings

File: src/views/Settings.vue [hunk 1]
```diff
@@ -2,42 +2,96 @@
...
+
+      <ion-card>
+        <ion-item lines="none">
+          <ion-label>Fulfillment: On</ion-label>
+        </ion-item>
+        <ion-item lines="none">
+          <ion-label class="text-wrap">Broadway has 47 outstanding orders and 77 in progress orders.</ion-label>
+        </ion-item>
+        <div class="actions">
+          <div class="desktop-only">
```
Comment (dt2patel) [inline src/views/Settings.vue:40]: move desktop only class to actions div instead of repeating it on children

**Review by dt2patel — CHANGES_REQUESTED** [review]
1 change.

File: src/views/Settings.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Settings.vue]: Done

**Review by dt2patel — APPROVED** [review]
looks good

**Pull request merged**